warp10/src/main/java/io/warp10/continuum/gts/MetadataTextComparator.java

```diff
@@ -70,10 +74,6 @@ public int compare(Metadata o1, Metadata o2) {
     // Names are identical, compare labels
     //
 
-    if (0 == o1.getLabelsSize() && 0 == o2.getLabelsSize()) {
-      return 0;
-    }
-
```

Comment:
If both size are zero, then it will return -1.

Reply:
If both size are zero, then it will check the attributes

Reply:
Are you sure?
```
    //
    // Names are identical, compare labels
    //
    int size1 = o1.getLabelsSize();
    int size2 = o2.getLabelsSize();

    if (0 == size1) {
      return -1;
    }
```

Reply:
Ok, got it. My bad…
Fixed